frontend/ui/data/creoptions.lua

```diff
@@ -56,6 +58,11 @@ local CreOptions = {
                 args = {0, 48, 96, 167, 212, 300},
                 event = "SetRenderDPI",
                 name_text_hold_callback = optionsutil.showValues,
+                help_text = _([[Sets the DPI used to scale CSS absolute units and images:
+- off: ignore absolute units (old engine behaviour).
+- 96¹’¹: at 96 dpi, 1 css pixel = 1 screen pixel; images are rendered as their original dimensions.
+- other values scale css absolute units and images by a factor (300 dpi = x3, 48 dpi = x0.5)
+Using your device's screen real DPI will make a css size of 1cm measures as 1cm.]]),
```

Comment:
Using your device's actual DPI will ensure 1cm in CSS actually means 1cm on screen.

Or something, just found that final sentence awkward ;).

Reply:
Or even "translates to", might be clearer.